src/styles/components/map-explorer.styl

```diff
@@ -0,0 +1,22 @@
+.map-explorer
```

Comment:
Just to double up on the suggestion, the grommet `Box` is a div by default with flex box properties set. A class here still might be necessary for scoping any children classes for the component.